src/components/common/Card.js

```diff
@@ -0,0 +1,13 @@
+import { Title } from "./Title_Styled";
+
+export function Card({ Image, title, subtitle = "", className, children }) {
+  return (
+    <div className={`${className} card-box`}>
+      {!!Image && Image}
```

Comment:
When rendering the value of a variable there is no need to check if the variable is truthy. `null`, `undefined` and `false` will be ignored and won't be rendered, you can read more [in the docs](https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored)

```suggestion
      {Image}
```